ubiquibot-config-default.json

```diff
@@ -84,6 +84,10 @@
       "name": "query",
       "enabled": false
     },
+    {
+      "name": "ask",
+      "enabled": true
```

Comment:
All commands should be `false` by default so that partners could opt-in for the features they need